crypto/init.c

```diff
@@ -44,8 +44,7 @@ static CRYPTO_ONCE base = CRYPTO_ONCE_STATIC_INIT;
 static int base_inited = 0;
 DEFINE_RUN_ONCE_STATIC(ossl_init_base)
 {
-    if (ossl_trace_init() == 0)
-        return 0;
+    /* no need to init trace */
```

Comment:
Note: adding a comment for something that has disappeared might raise more questions to a viewer in the future than it answers IMHO.

Reply:
I generally agree. 
Yet here the reader may wonder if initializing the trace API is needed since we're very early in the lib init and the trace functionality is used next.